uaa/uaa.yml

```diff
@@ -0,0 +1,121 @@
+database:
+    driverClassName: org.postgresql.Driver
+    url: jdbc:postgresql://postgres:5432/postgres
```

Comment:
should not be the url or the port coming from env variable ?

Reply:
Its linked to the docker container so will always be 5432, this file isn't parsed by docker-compose either so config values won't work